### ISSUE TITLE
no author affiliation with works/doi

### ISSUE THREAD

**Issue by animismus** [CrossRef/rest-api-doc#382]
when doing something like: 

    https://api.crossref.org/works/XXX/ 

where XXX is the doi, I don't get the authors affiliation. In fact this field comes back as an empty list ([]).

**Comment by pdavis8** [comment]
Hello, do you have a specific example of this? I suspect that it is not showing any data in that field as the metadata has not been sent to us by the member.

**Comment by animismus** [comment]
Yes. I guess you are right and that is the problem. I've checked two recent papers and both didn't have the affiliation by author. 
Another from 2015 had the information. Maybe it's a matter of publisher or do you think there might be a delay in sending this extra information?

What would you suggest for searching for publications by affiliation? I tried 

    /works?query.affiliation={AFFILIATION}&filter=from-pub-date:2018

but the results were very disappointing as I got a lot of unrelated affiliation results and, I think, only about 4 real results (should be around 20).

**Comment by VladimirAlexiev** [comment]
@animismus which affiliation (university) are you looking for?
Please note that funder name sometimes is also relevant: if a university funded some research, it's very likely one of the authors is affiliated to it.

**Comment by akhmerov** [comment]
I cannot find a member that *does* send the affiliation. I have checked publications from the last couple of years published by:

* American Association for the Advancement of Science (AAAS)
* IOP Publishing
* American Physical Society (APS)
* Springer Science and Business Media LLC

Also the [example from the readme](https://api.crossref.org/works/10.1037/0003-066X.59.1.29) doesn't list the affiliation.
